config/locales/forms.yml

```diff
@@ -227,6 +243,29 @@ en:
       publishers_job_listing_pay_package_form:
         benefits: Employee benefits
         salary_html: Salary (<span class='text-red'>Required</span>)
+      personal_details_form:
+        building_and_street: Building and street
+        email_address_html: Email address (<span class='text-red'>Required</span>)
+        first_name_html: First name (<span class='text-red'>Required</span>)
+        last_name_html: Last name (<span class='text-red'>Required</span>)
+        national_insurance_number_html: National insurance number (<span class='text-red'>Required</span>)
+        phone_number_html: Phone number (<span class='text-red'>Required</span>)
+        previous_names: Previous names
+        teacher_reference_number_html: Teacher reference number (TRN) (<span class='text-red'>Required</span>)
+        town_or_city: Town or city
+      subscription_form:
```

Comment:
I see these keys added here, but not removed from other places, why are you modifying `subscription_form` and `unsubscribe_feedback_form`?

Reply:
I've since removed it.